pymeasure/instruments/rohdeschwarz/sfm.py

```diff
@@ -0,0 +1,1501 @@
+        * Minimum 5 MHz
+        * Maximum 1 GHz
+        """,
+        validator = strict_range,
+        values = [5E6, 1E9]
+        )
+
+    rf_sweep_start = Instrument.control(
+        "SOUR:FREQ:STAR?",
+        "SOUR:FREQ:STAR %g",
+        """A float property controlling the start frequency for sweep in Hz
+
+        * Minimum 5 MHz
+        * Maximum 1 GHz
+        """,
+        validator = strict_range,
+        values = [5E6, 1E9]
+        )
+
+    rf_sweep_stop = Instrument.control(
+        "SOUR:FREQ:STOP?",
+        "SOUR:FREQ:STOP %g",
+        """A float property controlling the stop frequency for sweep in Hz
+
+        * Minimum 5 MHz
+        * Maximum 1 GHz
+        """,
+        validator = strict_range,
+        values = [5E6, 1E9]
+        )
+
+    rf_sweep_step = Instrument.control(
+        "SOUR:FREQ:STEP?",
+        "SOUR:FREQ:STEP %g",
+        """A float property controlling the stepwidth for sweep in Hz,
+
+        * Minimum 1 kHz
+        * Maximum 1 GHz
+        """,
+        validator = strict_range,
+        values = [1E3, 1E9]
+        )
+
+    rf_sweep_span = Instrument.control(
+        "SOUR:FREQ:SPAN?",
+        "SOUR:FREQ:SPAN %g",
+        """A float property controlling the sweep span in Hz,
+
+        * Minimum 1 kHz
+        * Maximum 1 GHz
+        """,
+        validator = strict_range,
+        values = [1E3, 1E9]
+        )
+
+    #Level (3.6.6.2)
+    #TODO POW-->POW[1..6]
+    level = Instrument.control(
+        "SOUR:POW:LEV?",
+        "SOUR:POW:LEV %g DBM",
+        """A float property controlling the output level in dBm,
+
+        * Minimum -99dBm
+        * Maximum 10dBm (depending on output mode)
+        """,
+        validator = strict_range,
+        values = [-99, 10],
+        )
+
+    #TODO POW-->POW[1..6]
+    level_mode = Instrument.control(
+        "SOUR:POW:LEV:MODE?",
+        "SOUR:POW:LEV:MODE %s",
+        """A string property controlling the output attenuator and linearity mode
+
+        Possible selections are:
+
+        ======  ====================  =================
+        Value   Meaning               max. output level
+        ======  ====================  =================
+        NORM    Normal mode           +6 dBm
+        LOWN    low noise mode        +10 dBm
+        CONT    continous mode        +10 dBm
+        LOWD    low distortion mode   +0 dBm
+        ======  ====================  =================
+
+        Contiuous mode allows up to 14 dB of level setting without use of the mechanical attenuator.
+
+        """,
+        validator = strict_discrete_set,
+        values = ["NORM","LOWN","CONT","LOWD"]
+        )
+
+    #TODO POW-->POW[1..6]
+    rf_out = Instrument.control(
+        "SOUR:POW:STAT?",
+        "SOUR:POW:STATE %s",
+        """ A bool property that controls the status of the RF-output,
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   RF-output disabled
+        True    RF-output enabled
```

Comment:
Call the property `rf_out_enabled` and you don't need to clarify with the table.

Reply:
Thanks for the proposal, property renamed and table in docstring removed.